src/components/title-block/TitleBlock.tsx

```diff
@@ -12,15 +13,21 @@ type TitleBlockProps = {
   img: string
   translationKey: string
   children?: React.ReactNode
+  style: SxProps
```

Comment:
Do it unimportant, cause you see warnings where you don't pass this prop.

Reply:
if won't define my type i will get error:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/89192246/61020814-bc94-4a7f-a938-42dc00697100)